lib/ui/viewer/gallery/collection_page.dart

```diff
@@ -57,38 +78,76 @@ class CollectionPage extends StatelessWidget {
       },
       reloadEvent: Bus.instance
           .on<CollectionUpdatedEvent>()
-          .where((event) => event.collectionID == c.collection.id),
+          .where((event) => event.collectionID == widget.c.collection.id),
       removalEventTypes: const {
         EventType.deletedFromRemote,
         EventType.deletedFromEverywhere,
         EventType.hide,
       },
-      tagPrefix: tagPrefix,
+      tagPrefix: widget.tagPrefix,
       selectedFiles: _selectedFiles,
       initialFiles: initialFiles,
-      albumName: c.collection.name,
+      albumName: widget.c.collection.name,
     );
     return Scaffold(
       appBar: PreferredSize(
         preferredSize: const Size.fromHeight(50.0),
         child: GalleryAppBarWidget(
-          appBarType,
-          c.collection.name,
+          widget.appBarType,
+          widget.c.collection.name,
           _selectedFiles,
-          collection: c.collection,
+          collection: widget.c.collection,
         ),
       ),
       body: Stack(
         alignment: Alignment.bottomCenter,
         children: [
           gallery,
-          GalleryOverlayWidget(
-            appBarType,
-            _selectedFiles,
-            collection: c.collection,
+          // GalleryOverlayWidget(
```

Comment:
We can remove this?

Reply:
I had left these intentionally, had mentioned in the description. 
"Note : There are some commented out lines of code, will clear them before merging to master. Need them for future context."